pkgs/applications/graphics/veusz/default.nix

```diff
@@ -0,0 +1,71 @@
+{ python3Packages
+, qtbase
+, ghostscript
+, wrapQtAppsHook
+, lib
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "veusz";
+  version = "3.3.1";
+
+  src = python3Packages.fetchPypi {
+    inherit pname version;
+    sha256 = "4ClgYwiU21wHDve2q9cItSAVb9hbR2F+fJc8znGI8OA=";
+  };
+
+  nativeBuildInputs = [ wrapQtAppsHook python3Packages.sip ];
+
+  buildInputs = [ qtbase ];
+
+  dontWrapQtApps = true;
+  preFixup = ''
+    wrapQtApp "$out/bin/veusz"
+  '';
+
+  # For some reason, if sip5 is found on the PATH, the option --sip-dir is
+  # ignored in setupPyBuildFlags, see
+  # https://github.com/veusz/veusz/blob/53b99dffa999f2bc41fdc5335d7797ae857c761f/pyqtdistutils.py#L292
+  patchPhase = ''
```

Comment:
```suggestion
  postPatch = ''
```

Reply:
I've added this change now. I guess it is better because one can still define patch files?

Reply:
Yes